run.py

```diff
@@ -164,7 +164,10 @@ def create_nodes(
         )
 
     log.info("Destroying existing osp instances..")
-    cleanup_ceph_nodes(osp_cred, instances_name)
+    for cluster in conf.get("globals"):
+        cluster_name = cluster.get("ceph-cluster").get("name", "ceph")
+        instance_name = f"{cluster_name}-{instances_name}-"
+        cleanup_ceph_nodes(osp_cred, instance_name)
```

Comment:
It is a good idea to be specific when we are attempting to redeploy the environment. 

However, this becomes a costly operation as we would be making 2x calls when we could have done with 2. Instead, we could opt to use `instance_name = f"-{instances_name}-"`

Reply:
Done. Modified